env/HERCULES.env

```diff
@@ -53,13 +53,13 @@ case ${step} in
     export wave_mpmd=${mpmd_opt}
 
  ;;
- "atmanlrun")
```

Comment:
CI test C96C48_ufs_hybatmDA failed on Hercules because `HERCULES.env` does not contain a section for `atmanlfv3inc`.  

@AndrewEichmann-NOAA , `ocnanalecen` is also missing from `HERCULES.env`.